packages/ts-config/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@dream-team/ts-config",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "files": [],
+  "publishConfig": {
+      "access": "public"
```

Comment:
We can make it private

Reply:
I believe it's unnecessery ❤️

Reply:
I think it should be handled separately on the Docker stage.